format_and_overlap_determination.md

```diff
@@ -0,0 +1,123 @@
+: A normalized enumeration. Description fields in the Hathifiles and submitted holdings are processed for enumeration and chronology. Only enumeration ends up being used. 
+
+
+## HT Item Format ##
+Every HT Item has one and only one format. This format is used to calculate the Cluster Format.
+
+**Serial**
+: HT Items with a **Bib Record ID** found on the **Print Serials List**, or in a **Large Cluster** 
+
+**Multi-part Monograph**
+: Is not a Serial. Shares a **Bib Record** with an HT Item that has a non-empty `n_enum`. [^1]
+
+**Single-part Monograph**
+: Is not a Serial or Multi-part
+
+**NB**: A cluster may contain HT Items of multiple formats.  
+**NB2**: Format as reported by members is irrelevant.
+
+## Cluster Format ##
+The Cluster Format is used in the overlap calculations. It is derived from the Item Formats of all the HT Items in the cluster, although it may not and often won't match all of the HT Item formats.
+
+**Multi-part Monograph**
+: Any of the HT Items in the cluster are MPMs.
+
+**SER / SPM**
+: Is not MPM. One or more of the HT Items in the cluster are SPM _and_ one or more of the HT Items in the cluster are SER.
+
+**SER**
+: Is not MPM or SER/SPM. Any of the HT Items in the cluster are SER (and by implication ALL are SER).
+
+**SPM**
+: Is not MPM, SER/SPM, or SER.
+
+## Cost Allocation ##
+
+**Public Domain vs In-Copyright**
+: For cost allocation purposes, HT Items with access "allow" in the Hathifiles are considered "public domain" and access "deny" are considered "in copyright". Rationale for the "allow" and "deny" found in the Hathifiles is underdefined. 
+
+**Target Cost**
+: Set by the Board and Executive Director. Portion of the budget to be paid for by HT Item allocations.
+
+**Cost Per Volume**
+: Target Cost / Total Number of HT Items
+
+**Member Weight**
+: A number assigned to a member by the ED that determines their share of Public Domain costs. 
+
+**Total Weight**
+: The sum of member weights found in the `ht_billing_members` table of the holdings database. **Which process/system has ownership of this table?**
+
+**Public Domain Cost**
+: Cost Per Volume * Number of Public Domain Volumes (see Public Domain vs In-Copyright)
+
+**Public Domain Cost for Member**
+: Total Public Domain Cost / Total Weight * Member Weight
+
+**Extra Per Member**
+: The IC Costs assigned to Hathitrust / Number of members found in the `ht_billing_members` table that aren't Hathitrust. Hathitrust has IC costs due to being the billing entity for some HT Items. This redistributes these costs across the membership.
+
+**HScore**
+: A share of an HT Item. The share is the inverse of the number of members who report holding the item.
+
+**Total HScore (for Member)**
+: Sum of HScores for Member for each of SPM, MPM and SER
+
+**Total IC Cost for Member**
+: Total HScore for Member * Cost Per Volume
+
+**Total Cost for Member**
+: Total IC Cost for Member + Total PD Cost for Member + Extra Per Member
+
+
+## HT Item Overlap ##
```

Comment:
In the legacy system, I think, we somehow allocate a share for each _copy_ of an item, if a member reports holding more than one copy?

Is that right and if so does the new system also do this?